popup.js

```diff
@@ -78,6 +78,18 @@ document.getElementById('convertto').onchange=function (e) {
 				}
 			}
 			fset.innerHTML = fdef;
+			var customFields = [...document.querySelectorAll('.slds-input')]
```

Comment:
this variable name is wrong. you are getting ALL the fields that have that class, the fixed fields also, so either change the name to something like `allFields` or set an attribute or a class on the custom fields and search on that to get only the custom fields.

Reply:
Yeah , i think, that is  is true, i am going to change it .